lib/ansible/modules/cloud/amazon/ec2_ami_copy.py

```diff
@@ -196,10 +210,11 @@ def main():
         kms_key_id=dict(type='str', required=False),
         wait=dict(type='bool', default=False),
         wait_timeout=dict(type='int', default=600),
-        tags=dict(type='dict')))
-
-    module = AnsibleModule(argument_spec=argument_spec)
+        tags=dict(type='dict')),
+        tag_equality=dict(type='bool', default=False))
 
+    module = AnsibleAWSModule(argument_spec=argument_spec)
+    # TODO: Check botocore version
     region, ec2_url, aws_connect_params = get_aws_connection_info(module, boto3=True)
     ec2 = boto3_conn(module, conn_type='client', resource='ec2', region=region, endpoint=ec2_url,
```

Comment:
Also, these 2 lines can be replaced with `ec2 = module.client('ec2')` and the boto3_conn and get_aws_connection_info imports can be removed.

Reply:
Done! Thanks.